helpers/tx.ts

```diff
@@ -0,0 +1,92 @@
+import axios from "axios";
+import moment from "moment";
+import ora from "ora";
+import WebSocket from "ws";
+
+const URL = {
+  API: "https://zetachain-athens.blockpi.network/lcd/v1/public/zeta-chain/crosschain",
```

Comment:
should this pull from a package like @zetachain/networks?

Reply:
Done: https://github.com/zeta-chain/toolkit/pull/10/commits/472f54f1f02499d7e729af4bc0e8301a36b3b02c

I can't publish an updated version of `@zetachain/publish` for some reason, though:

https://github.com/zeta-chain/networks/actions/runs/5518574540/jobs/10062714440

So for now this works only when linking the package directly.